client/web/src/response-handler.ts

```diff
@@ -55,12 +56,37 @@ function parseResponse(responseType: string, data: any): Response {
 			return newUpdateCanvas(data.UpdateCanvas);
 		case "ExportDocument":
 			return newExportDocument(data.ExportDocument);
+		case "UpdateWorkingColors":
+			return newUpdateWorkingColors(data.UpdateWorkingColors);
 		default:
 			throw new Error(`Unrecognized origin/responseType pair: ${origin}, ${responseType}`);
 	}
 }
 
-export type Response = SetActiveTool | UpdateCanvas | DocumentChanged | CollapseFolder | ExpandFolder;
+export type Response = SetActiveTool | UpdateCanvas | DocumentChanged | CollapseFolder | ExpandFolder | UpdateWorkingColors;
+
+export interface Color {
+	red: number;
+	green: number;
+	blue: number;
+	alpha: number;
+}
+
+export interface UpdateWorkingColors {
+	primary: Color;
+	secondary: Color;
+}
+
```

Comment:
Please remove this newline to stay with the convention in this file, which uses code blocks for exported functions and the same code block (no new line) for helper functions related to the exported one.